### PR TITLE
Update Web Bluetooth article for M56

### PR DESCRIPTION
**Pull request by beaufortfrancois**
R: @petele

### CODE REVIEW

File: src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md [hunk 1]
```diff
@@ -41,36 +39,26 @@ finalized yet, the Chrome Team is actively looking for enthusiastic developers
...
-in Chrome 56. In Chrome 55 or earlier, it can be enabled experimentally 
-on your origin in Origin Trials, or locally on your machine using an experimental flag.
-The implementation is partially complete and currently available on Chrome OS, 
-Chrome for Android M, Linux, and Mac.
-
-In Chrome 56 or later, go to `chrome://flags/#enable-experimental-web-platform-features` 
-otherwise go to `chrome://flags/#enable-web-bluetooth`, enable the highlighted flag,
-restart Chrome and you should be able to
+A subset of the Web Bluetooth API is available in Chrome 56 for Chrome OS,
+Chrome for Android M, and Mac. This means you should be able to
```
Comment (petele) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: Chrome for Android **M**?

Comment (beaufortfrancois) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: Yup.

Comment (petele) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: What is Chrome for Android M? I don't think I've heard that before?

**Comment by beaufortfrancois** [comment]
Android Marshmallow

**Comment by beaufortfrancois** [comment]
Should I say Android 6?

File: src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md (hunk 1 above)
Comment (petele) [inline src/content/en/updates/2015/07/interact-with-ble-devices-on-the-web.md]: Sorry - not enough caffeine yet. I totally read that wrong.

**Comment by beaufortfrancois** [comment]
@petele Can we push this live? I'm still seeing the old version at https://developers.google.com/web/updates/2015/07/interact-with-ble-devices-on-the-web

**Pull request merged**